fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -2563,6 +2563,7 @@ Bundle createSearchBundle(List<Resource> resources, FHIRSearchContext searchCont
                 Entry.Builder entryBuilder = Entry.builder();
                 if (resource != null) {
                     if (resource.getId() != null) {
+                        entryBuilder.id(resource.getId());
```

Comment:
why are we using the Entry.id for this?

Reply:
I guess its because there is no resource in the response entries, and so we can't use the normal `Entry.resource.id`.  Still, I think we might want a comment for this because using Entry.id for the id of the deleted resource is a bit unique.

Reply:
side-note:  you definitely didn't introduce this idea of using the entry id for this.  that comes from 
FHIRRestHelper.doDelete near line 722:
```
responseBundle.getEntry().stream().map(Bundle.Entry::getId).collect(Collectors.joining(","))
```